sci-libs/minc/minc-2.2.00.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Medical Imaging file format and Toolbox"
+HOMEPAGE="http://packages.bic.mni.mcgill.ca/tgz/${P}.tar.gz"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="BSD"
+
+DEPEND="sys-devel/flex
+	sys-devel/autoconf
+	sys-devel/gcc[cxx]
+	sys-libs/zlib
+	dev-perl/Text-Format"
+
+RDEPEND="dev-util/byacc
```

Comment:
I do not know, I just know that upstream lists it as a dep. Should I add it to DEPEND instead?